sdk/communication/communication-identity/test/public/communicationIdentityClient.mocked.spec.ts

```diff
@@ -76,4 +76,21 @@ describe("CommunicationIdentityClient [Mocked]", () => {
     assert.equal(newUser.communicationUserId, "identity");
     assert.isFalse("_response" in newUser);
   });
+
+  it("exchanges AAD token for ACS token", async () => {
+    const client = new TestCommunicationIdentityClient();
+    const spy = sinon.spy(exchangeAADtokenForACSTokenHttpClient, "sendRequest");
+    const response = await client.exchangeAADtokenForACStokenTest("AADtoken");
+
+    assert.equal(response.token, "token");
+    assert.equal(response.expiresOn.toDateString(), new Date("2011/11/30").toDateString());
+    sinon.assert.calledOnce(spy);
+  });
+
+  it("[exchangeAADtokenForACSToken] excludes _response from results", async () => {
+    const client = new TestCommunicationIdentityClient();
+    const response = await client.exchangeAADtokenForACStokenTest("AADtoken");
+
+    assert.isFalse("_response" in response);
```

Comment:
what is this test supposed to do? check that this field isn't set? could we use assert.notExists?

Reply:
Other operations have the same test so I just followed the pattern.
Looks like it checks that the raw response isn't exposed.